vignettes/configuration.Rmd

```diff
@@ -1,5 +1,5 @@
 ---
-title: "vcr configuration"
+title: "Tweak vcr default configuration"
```

Comment:
Could we use a different word than tweak? would "manage vcr configuration" be better?